src/testRunner.js

```diff
@@ -111,6 +111,26 @@ export default class TestRunner {
         });
     }
 
+    /**
+     * Show JSON-formatted metadata listing the tests and their test options,
+     * tags, etc.
+     *
+     * @param {RegExp} filter
+     */
+    testsMeta(filter) {
```

Comment:
Minor nitpick, but how would you feel about renaming this to something like `printMeta`? So it's clear that it doesn't return anything, it just prints to the console.